README.md

```diff
@@ -1,84 +1,55 @@
-# razorpay-php
+<p align="center"><img src="https://razorpay.com/images/logo.png"></p>
```

Comment:
The logo is for Razorpay, which still makes the README unclear. I don't like adding needless HTML where just markdown would suffice and be much cleaner. Maybe keep the title and badges on a single line like: https://github.com/shama/gaze?